src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -429,6 +459,9 @@ private static class EventIds
                 public static readonly EventId EndInvokeJSFailed = new EventId(111, "EndInvokeJSFailed");
                 public static readonly EventId EndInvokeJSSucceeded = new EventId(112, "EndInvokeJSSucceeded");
                 public static readonly EventId DispatchEventThroughJSInterop = new EventId(113, "DispatchEventThroughJSInterop");
+                public static readonly EventId LocationChanging = new EventId(114, "LocationChanging");
+                public static readonly EventId LocationChangingSucceded = new EventId(115, "LocationChangingSucceeded");
```

Comment:
`LocationChangeSucceeded` \ `LocationChangeFailed`? `LocationChangingSucceeded` rings a bit odd. Also `BeginLocationChange` instead of `LocationChanging`?

Reply:
LocationChange it is.